dockertest/environment.py

```diff
@@ -62,3 +65,13 @@ def get_selinux_context(path):
     """
     # First list item is null-terminated string length
     return selinux.getfilecon(path)[1]
+
+
+def selinux_is_enforcing():
+    """
+    Return True if selinux is currently in enforcing mode.
+    """
+    # Travis complains about missing selinux.ENFORCING
+    # pylint: disable=C0103
+    ENFORCING = 1  # Travis complains about missing selinux.ENFORCING
+    return selinux.security_getenforce() == ENFORCING
```

Comment:
So....you're proposing I re-write the entire thing because it's outputting enough emoji's in debug log?